src/pom.xml

```diff
@@ -652,6 +652,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.geoserver.community</groupId>
+        <artifactId>gs-graticule</artifactId>
+        <!--        <version>${gs.community.version}</version>-->
```

Comment:
I have merged https://github.com/camptocamp/geoserver-cloud-geoserver/pull/1, please update the version accordingly